tests/DemosTest.php

```diff
@@ -143,8 +143,8 @@ protected function getClient(): Client
             $localPath = static::ROOT_DIR . $request->getUri()->getPath();
 
             ob_start();
+            $app = $this->createTestingApp();
```

Comment:
this must be kept in try, otherwise try/finally for buffering is not atomic

Reply:
later i need to call `$app->getResponse()' to check it, i do it only to avoid errors in static code analysis, how to instruct phpstan that is not an error?

Reply:
then you do something wrong ;-)

Reply:
> then you do something wrong ;-)
this time I don't think so, I move back in try/catch, phpstan will fail. anyway i can added 2 `@phpstan-ignore-line`

Reply:
Fixed